packages/zoe/CONTRIBUTING.md

```diff
@@ -46,4 +46,56 @@ Before submitting a pull request, please:
 * `git tag -a zoe-v$VERSION -m "zoe-v$VERSION"`
 * `yarn publish --access public`
 * `git push`
-* `git push --tags`
+* `git push origin zoe-v$VERSION`
+
+Then, once the release has been made, the packages dependent on Zoe
+should be updated in a PR reviewed by the owners of the packages.
+Those packages are:
+* packages/agoric-cli (`packages/agoric-cli/template/contract/package.json`)
+* packages/cosmic-swingset (`packages/cosmic-swingset/package.json`)
+
+To test that that the Zoe update works (in the most basic sense) with cosmic-swingset, do:
+
+1. Run `yarn install` from the workspace root
+2. Run `yarn build` from the workspace root
+3. `cd packages/cosmic-swingset`
```

Comment:
Arguably, you could just test `agoric-cli` as long as you use the `--sdk` option, and when you run `agoric --sdk start` that gives you a working scenario3.  Testing the deploy scripts can be a separate step.

But this is fine for now.  We can refine the steps once `agoric-cli` is fully working.

Reply:
Good point